README.md

```diff
@@ -1 +1,82 @@
+[![Coverage Status](https://coveralls.io/repos/github/andela-hchukwu/Document-Management-System/badge.svg?branch=chore%2F142312565%2Fcoveralls-code-coverage-service)](https://coveralls.io/github/andela-hchukwu/Document-Management-System?branch=chore%2F142312565%2Fcoveralls-code-coverage-service)
+[![Code Climate](https://codeclimate.com/github/andela-hchukwu/Document-Management-System/badges/gpa.svg)](https://codeclimate.com/github/andela-hchukwu/Document-Management-System)
+
+# Document-Management-System
+
+This is a system (API) that manages documents with users and user roles. Each document defines access rights; the document defines which roles can access it. Also, each document specifies the date it was published. It is built with NodeJS, Express and Postgres as it's database.
+Source code employs ES6 syntax traspiled down to ES5 using Babel.
+
+### Technologies Used
+---
+- JavaScript (ES6)
+- Node.js
+- Express
+- Postgresql
+- React/Redux
+- Sequelize ORM.
+
+## Development
+This application was developed using [NodeJs](https://nodejs.org) with express for routing. Postgres was used for persisting data with [Sequelize](https://sequelizejs.org) as [ORM](https://en.wikipedia.org/wiki/Object-relational_mapping).
+
+The frontend was built with the [react](https://facebook.github.io/react/) and [redux](reduxjs.org) framework.
+
+
+### Installation
+---
+
+- Clone the project repository.
+- Run git clone https://github.com/andela-hchukwu/Document-Management-System.git.
+- Change directory into the Document-Management-System directory.
+- Run npm install to install the dependencies in the package.json file.
+- Use Postman or any API testing tool of your choice to access the endpoints.
+
+## Usage
+- Run DB migrate commmand with `sequelize db:migrate`.
+- Run DB seeder command with `sequelize db:seed` to seed initial data into your DB.
+- Start the app with `$ npm start`
+- Login, Sign Up and start creating Documents once the app opens up on the browser
+
+#### Key Application Features
+A created user will have a role: admin, regular, rated/restricted, children.
+- Regular, rated/restricted and children Users can:
+    - Create an account
+    - Login
+    - Create a document
+    - Limit access to a document by specifying an access group `i.e. public, private or role`.
+    - View public documents created by other users.
+    - View documents created by their access group with access level set as `role`.
+    - Edit already created documents.
+    - View `public` and `role` access level documents of other regular users.
+    - Logout.
+
+- In addition to the general user functions, an admin user can:
+    - View all users.
+    - View all created documents.
+    - Delete any user.
+    - Update any user's records.
+    - Create a new role.
+    - View all created roles.
+    - Delete created roles aside the default roles `admin` and `regular`
+
+
+**Documents**:
+Documents can be created and must have:
+- title
```

Comment:
your git readme file needs to show  a list of all the endpoints and their output.